docs/reference/commandline/build.md

```diff
@@ -96,22 +96,22 @@ For example, run this command to use a directory called `docker` in the branch
 `container`:
 
 ```bash
-$ docker build https://github.com/docker/rootfs.git#container:docker
+$ docker build https://github.com/docker/rootfs#container:docker
```

Comment:
There were some concerns about the "remote" functionality in general (separate discussion), e.g. giving a client permission to have the remote server use keys that happen to be present on the server's root home (that's for `git@` notation)

So not entirely sure we should automatically do that

/cc @tonistiigi